TACC/protx-dashboard
# Task/COOKS-327: Refactor maltreatment route to not use selected area

## Overview: ##
Refactor maltreatment route to use Geoid instead of selectedArea

## Related Jira tickets: ##

* [COOKS-123](https://jira.tacc.utexas.edu/browse/COOKS-123)

## Summary of Changes: ##
Update frontend and backend so that the maltreatment chart data is obtained using Geoid instead of selectedArea.

## Testing Steps: ##
1. Go to https://cep.test/protx/dash/maltreatment
2. Click on the map and ensure that the correct chart is displayed on the right. UI photos below capture before and after to confirm that the correct chart data is displayed (i.e. no change before and after).


## UI Photos:
**Before**
![Screen Shot 2022-10-17 at 12 44 57 PM](https://user-images.githubusercontent.com/79928051/196246600-a099d1e8-bbf7-4ca3-9d93-8cd733cba125.png)

**After**
![Screen Shot 2022-10-17 at 12 39 07 PM](https://user-images.githubusercontent.com/79928051/196246099-566e4ab6-33d0-465f-a551-18cbf87a283a.png)

## Notes: ##
